### PR TITLE
Subsystems Log Path -> master

### PR DESCRIPTION
**Pull request by NoamRosenberg08**

### CODE REVIEW

**Review by tnrz135 — DISMISSED** [review]

File: src/main/java/frc/utils/GBSubsystem.java [hunk 1]
```diff
@@ -6,6 +6,12 @@
...
 
+    private final String loggingPath;
+
+    public GBSubsystem(String loggingPath) {
+        this.loggingPath = "Subsystems/" + loggingPath;
+    }
+
     @Override
     public final void periodic() {
         Logger.recordOutput(getLogPath() + "Current Command", getCurrentCommandName());
```
Comment (YoniKiriaty) [inline src/main/java/frc/utils/GBSubsystem.java:17]: Why are you doing getLogPath if loggingPath is already a variable in this class?

Comment (YHerm) [reply src/main/java/frc/utils/GBSubsystem.java:17]: We can do public final but its ugly

Comment (YoniKiriaty) [reply src/main/java/frc/utils/GBSubsystem.java:17]: I don't want you to switch to public final, but you can just use loggingPath

Comment (YoniKiriaty) [reply src/main/java/frc/utils/GBSubsystem.java:17]: I meant only in this line change it

File: src/main/java/frc/utils/GBSubsystem.java [hunk 2]
```diff
@@ -6,6 +6,12 @@
 
 public abstract class GBSubsystem extends SubsystemBase {
 
+    private final String logPath;
+
+    public GBSubsystem(String logPath) {
+        this.logPath = "Subsystems/" + logPath;
```
Comment (yonnbro) [inline src/main/java/frc/utils/GBSubsystem.java:12]: why remove from constants?

**Review by yonnbro — CHANGES_REQUESTED** [review]

**Review by bossyshrimp68 — APPROVED** [review]

**Pull request merged**